client/components/token-field/index.jsx

```diff
@@ -476,6 +508,18 @@ var TokenField = React.createClass( {
 		}
 	},
 
+	_valueContainsToken( token ) {
+		let search = find( this.props.value, ( item ) => {
```

Comment:
Can the `token` argument here be an object itself? If not, is this inconsistency confusing?